### PR TITLE
fix the problem abut login button.

### PR DESCRIPTION
**Pull request by MSMingzhou**

### CODE REVIEW

File: app/views/security/enable.html.erb [hunk 1]
```diff
@@ -1,36 +1,71 @@
...
+            </div>
+          </div>
         </div>
-      <% end %>
+      </div>
     </div>
   </div>
+  <% else %>
+
+    <h2>You are not signed in.</h2>
```
Comment (shiqicao) [inline app/views/security/enable.html.erb]: localization?

File: app/views/security/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                  </div>
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+  <% else %>
+
+      <h2>You are not signed in.</h2>
```
Comment (shiqicao) [inline app/views/security/index.html.erb]: localization?

File: app/views/devise/passwords/edit.html.erb [hunk 3]
```diff
@@ -1,48 +1,38 @@
...
-  </div>
+  <div class="mask-content">
+    <div class="register-container">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">RESET YOUR PASSWORD</h3>
+        </div>
+        <div class="panel-body">
+          <div class="reset-password">
+          <%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
```
Comment (jordansaints) [inline app/views/devise/passwords/edit.html.erb]: indentation is off by one in this block (the one starting on line 11)

File: app/views/devise/passwords/edit.html.erb [hunk 4]
```diff
@@ -1,48 +1,38 @@
...
-  <div class="col-xs-12 col-sm-11 col-md-6 mb-2">
-    <div class="text-center">
-      <h2 class="title">Change your password</h2>
-    </div>
-  </div>
+  <div class="mask-content">
+    <div class="register-container">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title">RESET YOUR PASSWORD</h3>
```
Comment (jordansaints) [inline app/views/devise/passwords/edit.html.erb]: Sentence case? `Reset your password`?

File: app/views/devise/passwords/edit.html.erb [hunk 5]
```diff
@@ -1,48 +1,38 @@
...
 
-      <div class="form-group">
-        <%= f.label :password, 'New password' %><br />
-        <% if @minimum_password_length %>
-          <em>(<%= @minimum_password_length %> characters minimum)</em><br />
-        <% end %>
-        <%= f.password_field :password, autofocus: true, autocomplete: 'off', class: 'form-control form-control-lg underlined' %>
-      </div>
+            <div class="form-group">
+              <%= f.label :password_confirmation, (t 'password.confirm_new_password')  ,for: 'inputEmail1'%><br />
```
Comment (jordansaints) [inline app/views/devise/passwords/edit.html.erb]: nitpick: odd spacing around the comma separators; should be `(t 'password.confirm_new_password'), for:`

File: app/views/devise/passwords/edit.html.erb [hunk 6]
```diff
@@ -1,48 +1,38 @@
...
+              <%= f.password_field :password_confirmation, id: 'inputEmail1',autocomplete: 'off', class: 'form-control', placeholder: (t 'password.placeholder_confirm_new_password')  %>
```
Comment (jordansaints) [inline app/views/devise/passwords/edit.html.erb]: param spacing around `autocomplete`

File: app/views/devise/passwords/edit.html.erb [hunk 7]
```diff
@@ -1,48 +1,38 @@
...
-      <div class="actions">
-        <%= f.submit 'Change my password', class: 'btn btn-block btn-success' %>
       </div>
-    <% end %>
+    </div>
   </div>
 </div>
-<% end %>
\ No newline at end of file
+<% end %>ß
```
Comment (jordansaints) [inline app/views/devise/passwords/edit.html.erb]: extra character at the end of this line

File: app/views/devise/registrations/edit.html.erb [hunk 8]
```diff
@@ -1,40 +1,60 @@
...
-          <%= f.label :password_confirmation %>
-          <%= f.password_field :password_confirmation, autocomplete: 'off',  class: "px-1 form-control form-control-lg underlined", placeholder: 'Enter new password' %>
-      </div>
+                <div class="form-group">
+                    <%= f.label :password_confirmation %>
+                    <%= f.password_field :password_confirmation, autocomplete: 'off',  class: "form-control", placeholder: 'Enter new password' %>
+                </div>
+
+                <% if @otp_enabled %>
+                <div class="form-group">
```
Comment (jordansaints) [inline app/views/devise/registrations/edit.html.erb]: indent block

File: app/views/devise/registrations/edit.html.erb [hunk 9]
```diff
@@ -1,40 +1,60 @@
...
+                </div>
+                <% end %>
 
-      <div class="d-flex justify-content-between mt-4">
-        <%= f.submit  "CHANGE", class: "btn btn-primary btn-block btn-success btn-phone col-6" %>
+                <div class="form-group google-unbind-btn">
+                  <%= f.submit  "SUBMIT", class: "btn btn-primary btn-block" %>
+                </div>
+              <% end %>
+              </div>
```
Comment (jordansaints) [inline app/views/devise/registrations/edit.html.erb]: indentation off with above block

File: app/views/index/index.html.erb [hunk 10]
```diff
@@ -1,44 +1,63 @@
...
-    <div class="col-12">
-      <div class="text-center">
-        <div class="wrap">
-          <h2 class="wrap">Signed in as <%= current_account.email %></h2><br/>
+      <div class="flex-content">
+        <div class="profile-content">
+          <div class="overview-index">
+            <div class="row">
+              <div class="col-md-3 text-center">
+                <h3> <%= current_account.email %></h3>
```
Comment (jordansaints) [inline app/views/index/index.html.erb]: extra space here

File: app/views/layouts/shared/_overview.html.erb [hunk 11]
```diff
@@ -0,0 +1,24 @@
+<div class="row profile-head">
+  <div class="col-md-4">
+    <div class="profile-head-item estimated">
+      <h5>ESTIMATED VALUE</h5>
+      <div><strong>24</strong><small>BTC</small></div>
+      <p>&asymp; 8000.00 USD</p>
```
Comment (jordansaints) [inline app/views/layouts/shared/_overview.html.erb]: these are all temp/test values right?

File: app/views/profiles/index.html.erb [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+                    <div><strong>Document</strong></div>
+                  </a>
+                  <p>Your document is being reviewed</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+  <% else %>
```
Comment (jordansaints) [inline app/views/profiles/index.html.erb]: indentation off

**Pull request merged**